modules/install/mod_settings/config.xml

```diff
@@ -413,7 +413,20 @@ nospamfor.us</default>
         <name>google_analytics_id</name>
         <type>string</type>
         <default></default>
-        <description>Google-Analytics-ID. ID um Google-Analytics Code zu aktivieren</description>
+        <description>Google API-Key um Google-Analytics Code zu aktivieren</description>
+      </item>
+        <item>
+        <name>google_maps_key</name>
+        <type>string</type>
+        <default></default>
+        <description>Google API-Key um Google-Maps Code zu aktivieren</description>
+      </item>
+        </item>
+        <item>
+        <name>google_translate_key</name>
```

Comment:
Where is this used? This seem to be added, but not used.

Reply:
Yes. related translation code is defunct (see #842) and will need an API key to work again.
I thought to add that already, but it can also be left out and included in the PR to fix that

Reply:
I prefer to bundle the translation API key in the related change on #842.

Reply:
Removed together with the related documentation.
There was a duplication of `</item>`, so it was good to touch that file again anyways